arm-iothub/2017-01-19/swagger/iothub.json

```diff
@@ -964,9 +964,451 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/certificates": {
+      "get": {
+        "tags": [ "GET" ],
+        "summary": "Get the certificate list.",
+        "description": "Get the certificate list.",
+        "operationId": "Certificates_Get",
```

Comment:
operationId = IotHubResource_ListCertificates for consistency? similar to IotHubResource_ListKeys?